### ISSUE TITLE
Special character in PowerShell stdout/stderr causing playbook to fail

### ISSUE THREAD

**Issue by tianweiliu** [ansible/ansible#12897]
Issue Type: Bug Report
Ansible Version: 1.9.4
Ansible Configuration: Fresh homebrew installation
Environment: Mac OS X 10.10.4

Summary:

Special character in stdout produced by running `npm install` in PowerShell will result in following error causing playbook to stop, although the `npm install` did run without error:

```
TASK: [npm install] *********************************************************** 
fatal: [10.211.55.8] => Traceback (most recent call last):
  File "/usr/local/Cellar/ansible/1.9.4/libexec/lib/python2.7/site-packages/ansible/runner/__init__.py", line 586, in _executor
    exec_rc = self._executor_internal(host, new_stdin)
  File "/usr/local/Cellar/ansible/1.9.4/libexec/lib/python2.7/site-packages/ansible/runner/__init__.py", line 789, in _executor_internal
    return self._executor_internal_inner(host, self.module_name, self.module_args, inject, port, complex_args=complex_args)
  File "/usr/local/Cellar/ansible/1.9.4/libexec/lib/python2.7/site-packages/ansible/runner/__init__.py", line 1036, in _executor_internal_inner
    result = handler.run(conn, tmp, module_name, module_args, inject, complex_args)
  File "/usr/local/Cellar/ansible/1.9.4/libexec/lib/python2.7/site-packages/ansible/runner/action_plugins/script.py", line 128, in run
    result = handler.run(conn, tmp, 'raw', module_args, inject)
  File "/usr/local/Cellar/ansible/1.9.4/libexec/lib/python2.7/site-packages/ansible/runner/action_plugins/raw.py", line 47, in run
    become=self.runner.become)
  File "/usr/local/Cellar/ansible/1.9.4/libexec/lib/python2.7/site-packages/ansible/runner/__init__.py", line 1174, in _low_level_exec_command
    in_data=in_data)
  File "/usr/local/Cellar/ansible/1.9.4/libexec/lib/python2.7/site-packages/ansible/runner/connection_plugins/winrm.py", line 165, in exec_command
    return (result.status_code, '', unicode(result.std_out).encode('utf-8'), unicode(result.std_err).encode('utf-8'))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 203: ordinal not in range(128)
```

Steps To Reproduce:

Playbook:

```
...
  tasks:
    - name: npm install
      script: powershell/npm_install.ps1
...
```

PowerShell script:

```
cd C:\my_node_app\
npm install
```

**Comment by abadger** [comment]
This should be fixed in 2.0 via this commit: c845181dc1177ec61d10cf051e1f6ff24074b79b  and for 1.9.x (if there's a 1.9.5) in 4472889632c40d64e6fb4d7a1002caa4e64328e0  You should be able to apply the latter commit to your 1.9.4 install to get things working.

**Comment by abadger** [comment]
# Closing This Ticket

Hi!

We believe recent commits (likely detailed above) should resolve this question or problem for you.

This will also be included in the next major release.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!
